### PR TITLE
format: auto formating

### PR DESCRIPTION
**Pull request by Purevdorj-Batgerel**
auto formating async folder

### CODE REVIEW

File: async/setTimeout-let-var.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ for (var i = 0; i < 10; i++) {
   setTimeout(() => {
     console.log(i);
   }, 1000);
-  setTimeout(function() {
+  setTimeout(function () {
```
Comment (angarag) [inline async/setTimeout-let-var.js:5]: ok

File: async/setTimeout-let-var.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ for (let i = 0; i < 10; i++) {
...
```
Comment (angarag) [inline async/setTimeout-let-var.js:14]: oki

Comment (Purevdorj-Batgerel) [reply async/setTimeout-let-var.js:14]: ok

**Comment by angarag** [comment]
reopened

**Pull request merged**